Objects/Objects/BuiltElements/Space.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Objects.Geometry;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Objects.BuiltElements.Revit.Curve;
+
+namespace Objects.BuiltElements
+{
+    public class Space : Base, IHasArea, IHasVolume, IDisplayMesh
+    {
+        public string name { get; set; }
+        public string number { get; set; }
+        public double area { get; set; }
+        public double volume { get; set; }
+        public Point basePoint { get; set; }
+        public Level level { get; set; }
+        public double baseOffset { get; set; } = 0;
+        public Level upperLimit { get; set; }
+        public double limitOffset { get; set; } = 0; 
+        public ICurve boundary { get; set; }
+        public string spaceType { get; set; }
+        public string zoneId { get; set; } 
```

Comment:
What is the ZoneId? In general, we want to avoid sending Elemenet Ids since they will vary from revit document to document unless they are some sort of fixed ids. 
A better way to support zones could be to pass their elmenent name and then try to match it when receiving.

Reply:
ZoneId is meant for the element id of the Zone which contains the current Space - have now updated to use that element name rather than id to keep track of this info.

A related item - do you want to just add a `Zone` object to the BuiltElements? Would you prefer to just have this object included as a detachable property in the `Space`?

Reply:
I think it's fine like this for now! We might decide to do what you suggest later if needed and deprecate this prop.